modules/openy_features/openy_location/modules/openy_loc_filter/openy_loc_filter.install

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @file
+ * Open Y Location filter install file.
+ */
+
+use Drupal\node\NodeInterface;
+
+/**
+ * Set default locations.
+ */
+function openy_loc_filter_install() {
+  $config = \Drupal::configFactory()
+    ->getEditable('openy_loc_filter.location_filter_settings');
+
+  /** @var \Drupal\Core\Database\Query\SelectInterface $query */
+  $query = \Drupal::database()->select('node_field_data', 'n')
+    ->fields('n', ['nid', 'title', 'type'])
+    ->condition('type', ['branch', 'camp'], 'IN')
+    ->condition('status', NodeInterface::PUBLISHED);
+  $items = $query->execute()->fetchAll();
+
+  $locations = [];
+  foreach ($items as $item) {
+    $locations['All'] = t('All');
```

Comment:
@AleksandrPrykhodkoDev Do we really need to add `'All'` element each iteration? You can add it only once before the loop.